packages/frontend/README.md

```diff
@@ -1 +1,48 @@
-# RAIO frontend
+# Rafiki Admin
+
+## Development
+
+This project assumes that you have a local Rafiki backend instance up and running. See the `Environment Setup` and `Local Development` sections in the main [README](../../README.md) and the `Local Playground` section in the infrastructure/local [README](../../infrastructure/local/README.md).
+
+To start the project in development mode, we first need to generate the GraphQL types for our queries and mutations.
+
+To generate the GraphQL types file run:
+
+```sh
+pnpm --filter backend generate
+```
+
+To start the application run:
+
+```sh
+pnpm --filter frontend dev
+```
+
+Now you can access the application on [http://localhost:3005](http://localhost:3005).
```

Comment:
Maybe it's worth specifying its for `http://localhost:3001/graphql` admin GraphQL (cloud-nine-wallet) until we have 
https://github.com/interledger/rafiki/pull/969/files#diff-807bf94fbfe6e1750d65f29d075a5d6553c8ce1813cee64b3a3e9c0c2e006ef8R30

the port in a configuration file. Or maybe it should be pulled out into an env variable during startup?

We could also run two admin UIs, one for each of the respective localenv wallets? 
CC @sabineschaller